pkg/lru/test_expirable_lru.go

```diff
@@ -0,0 +1,134 @@
+package lru
+
+import (
+	"testing"
+	"time"
+)
+
```

Comment:
Is this test copied or created from scratch?

Reply:
This file is a utility for testing purposes that was created from scratch.